backend/app/tests/crud/evaluations/test_cron.py

```
@@ -0,0 +1,349 @@
+
+import pytest
+from sqlmodel import Session, select
+
+from app.crud.evaluations.cron import (
+    process_all_pending_evaluations,
+    process_all_pending_evaluations_sync,
+)
+from app.crud.evaluations.core import create_evaluation_run
+from app.core.util import now
+from app.models import Organization, Project
+from app.models import BatchJob, Organization, Project
```

Ruff (0.14.8)

Redefinition of unused `Organization` from line 12

Remove definition: `Organization`

(F811)